eng/common/build.sh

```diff
@@ -105,7 +105,7 @@ while [[ $# > 0 ]]; do
     -binarylog|-bl)
       binary_log=true
       ;;
-    -excludecibinarylog|-nobl)
+    -excludeCIBinarylog|-nobl)
```

Comment:
I don't think this is correct either, all `$opt` should be in the lower case

Reply:
Given this is part of eng\common and controlled by arcade, I'll leave as is.